docs/01-overview/02-main-areas/service-management/smgt-12-hb-bind-addons.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Bind addons
+---
+
+If you defined in the `meta.yaml` file that your plan is bindable, you must also create a `bind.yaml` file.
+The `bind.yaml` file supports the Service Catalog binding concept. It is mandatory for all bindable plans as it contains information needed during the binding process. Currently, Kyma supports only the [credentials-type binding](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#types-of-binding).   
+
+>**NOTE:** Resolving the values from the `bind.yaml` file is a post-provision action. If this operation ends with an error, the provisioning also fails.
+
+In the `bind.yaml` file, you can use the Helm chart templates directives. See the example:
+
+```yaml
+# bind.yaml
+credential:
+  - name: HOST
+    value: {{ template "redis.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+{{- if .Values.usePassword }}
+  - name: REDIS_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "redis.fullname" . }}
+        key: redis-password
+{{- end }}
+```
+In this example, the system renders the `bind.yaml` file. The system resolves all the directives enclosed in the double curly braces in the same way as in the files located in the `templates` directory in your Helm chart.
+
+
+## File specification
+
```

Comment:
aren't those tables more a kind of technical reference information that users look up occasionally?

Reply:
Same as above